core/templates/dev/head/components/ck-editor-helpers/ck-editor-5-rte.directive.ts

```diff
@@ -32,22 +32,28 @@ oppia.directive('ckEditor5Rte', [
       scope: {
         uiConfig: '&'
       },
-      template: '<div><div></div>' +
-                '<div contenteditable="true" class="oppia-rte">' +
-                '</div></div>',
-      require: '?ngModel',
+      template: '<div>' +
+                '<div></div>' +
+                '<div contenteditable="true" class="oppia-rte"></div>' +
+                '</div>',
+      require: '^ngModel',
 
-      link: function(scope: ICustomScope, el, attr, ngModel) {
+      link: function(scope: ICustomScope, elem, attrs, ngModel) {
         var _RICH_TEXT_COMPONENTS = RteHelperService.getRichTextComponents();
         var names = [];
         var icons = [];
-        var canUseFs = ContextService.getPageContext() ===
-          PAGE_CONTEXT.EXPLORATION_EDITOR;
+        var canUseFs = (
+          ContextService.getPageContext() === PAGE_CONTEXT.EXPLORATION_EDITOR ||
+          ContextService.getPageContext() === PAGE_CONTEXT.TOPIC_EDITOR ||
+          ContextService.getPageContext() === PAGE_CONTEXT.STORY_EDITOR ||
+          ContextService.getPageContext() === PAGE_CONTEXT.SKILL_EDITOR);
+
         _RICH_TEXT_COMPONENTS.forEach(function(componentDefn) {
+         var componentRequiresFsButFsCannotBeUsed =
+        !canUseFs && componentDefn.requiresFs;
```

Comment:
Indent this by 2 from the previous line. For clarity, use parens. I.e.:
```
        var componentRequiresFsButFsCannotBeUsed = (
          !canUseFs && componentDefn.requiresFs);
```

Reply:
Done PTAL Thanks!